packages/generator/test/gulpfile-helpers.test.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as diffProcessor from "@commerce-apps/raml-toolkit";
+import { NodeDiff } from "@commerce-apps/raml-toolkit";
+import { diffNewAndArchivedRamlFiles, RamlDiff } from "../src/gulpfileHelpers";
+import { generatorLogger } from "../src/logger";
+
+import _ from "lodash";
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+import fs from "fs-extra";
+import path from "path";
+import sinon from "sinon";
+import tmp from "tmp";
+
+const expect = chai.expect;
+let tmpDir: tmp.DirResult;
+
+before(() => {
+  chai.should();
+  chai.use(chaiAsPromised);
+});
+
+describe("diffNewAndArchivedRamlFiles", () => {
+  let apiConfig;
+  let apiConfigFile;
+  let diffRamlStub: sinon.SinonStub;
+  const nodeDiffArr = [];
+
+  before(() => {
+    apiConfig = {
+      family1: [
+        { assetId: "api1", fatRaml: { mainFile: "api1.raml" } },
+        { assetId: "api2", fatRaml: { mainFile: "api2.raml" } }
+      ],
+      family2: [
+        { assetId: "api3", fatRaml: { mainFile: "api3.raml" } },
+        { assetId: "api4", fatRaml: { mainFile: "api4.raml" } }
+      ]
+    };
+    nodeDiffArr.push(new NodeDiff("#/web-api/end-points/test-endpoint"));
```

Comment:
```suggestion
    nodeDiffArr.push(new NodeDiff("#/web-api/endpoints/test-endpoint"));
```
Choose `endpoint[s]` or `end-point[s]`, not both. 😛